internal/support/trace_parser.py

```diff
@@ -469,6 +469,22 @@ def ProcessTimelineEvents(self):
                 self.interactive.append([int(math.ceil(self.interactive_start / 1000.0)),
                                          int(math.floor(self.interactive_end / 1000.0))])
 
+            # Extend last interactive window to the end of the run
+            # Assuming that if there are no events past the last interactive window
+            # then the page was interactive for that time
+            if self.cpu['total_usecs'] == self.interactive_end and run_time > 0 and \
```

Comment:
Just for my sanity, can we only extend the run time if it is larger than the captured end time?  I assume it will always be the case but in case weird clock things happen I don't want to be adding a bad window to the end.